blackbox/docs/general/dql/joins.rst

```diff
@@ -216,10 +216,10 @@ in the left table.
 
 This is the default algorithm used for all types of joins.
 
-Block Hash Join Algorithm
+Block hash join algorithm
 .........................
 
-The performance of `Equi-Joins`_  is substantially improved by using the
+The performance of `Equi-Joins`_ is substantially improved by using the
```

Comment:
revert

Reply:
re-insert second blank space? done.

Reply:
the two spaces is an error. but it should be fixed in a separate PR